### PR TITLE
Optimize some codes

### PR DESCRIPTION
**Pull request by AngLi2**

### CODE REVIEW

File: spring-cloud-huawei-common/src/main/java/com/huaweicloud/common/util/URLUtil.java [hunk 1]
```diff
@@ -45,7 +46,7 @@
    */
   public static String[] splitIpPort(String url) {
     if (url == null) {
-      return null;
+      return new String[0];
     }
```
[the remark this answers is not in the document]
Comment (AngLi2) [reply spring-cloud-huawei-common/src/main/java/com/huaweicloud/common/util/URLUtil.java]: done

File: spring-cloud-huawei-common/src/main/java/com/huaweicloud/common/util/URLUtil.java [hunk 2]
```diff
@@ -125,45 +126,38 @@ private static String removeSlash(String url) {
...
-    }
-    return StringUtils.isEmpty(sysURL) ? dealMutiUrl(envURL) : dealMutiUrl(sysURL);
+    return getStrings(SYSTEM_KEY_CONFIG_CENTER);
   }
 
   public static List<String> getEnvServerURL() {
+    return getStrings(SYSTEM_KEY_SERVICE_CENTER);
+  }
+
+  private static List<String> getStrings(String systemServer) {
```
[the remark this answers is not in the document]
Comment (AngLi2) [reply spring-cloud-huawei-common/src/main/java/com/huaweicloud/common/util/URLUtil.java]: done

File: spring-cloud-huawei-servicecomb-discovery/src/main/java/com/huaweicloud/servicecomb/discovery/discovery/ServiceCombDiscoveryClient.java [hunk 3]
```diff
@@ -73,6 +74,6 @@ public String description() {
     } catch (ServiceCombException e) {
       LOGGER.error("getServices failed", e);
     }
-    return null;
+    return Collections.emptyList();
   }
```
[the remark this answers is not in the document]
Comment (AngLi2) [reply spring-cloud-huawei-servicecomb-discovery/src/main/java/com/huaweicloud/servicecomb/discovery/discovery/ServiceCombDiscoveryClient.java]: done

File: spring-cloud-huawei-swagger/src/main/java/com/huaweicloud/swagger/ServiceCombSwaggerHandlerImpl.java [hunk 4]
```diff
@@ -113,19 +114,18 @@ public void init(String appName, String serviceName) {
...
-              api.setProduces(contentTypeList);
-              api.setTags(Arrays.asList(className));
-            });
-          });
+          temSwagger.getPaths().forEach((path, operation) ->
+              operation.getOperations().forEach(api -> {
+                api.setConsumes(contentTypeList);
+                api.setProduces(contentTypeList);
+                api.setTags(Arrays.asList(className));
+              }));
```
Comment (GuoYL123) [inline spring-cloud-huawei-swagger/src/main/java/com/huaweicloud/swagger/ServiceCombSwaggerHandlerImpl.java]: It seems not change. Is it necessary to delete {} ?

File: spring-cloud-huawei-swagger/src/main/java/com/huaweicloud/swagger/ServiceCombSwaggerHandlerImpl.java [hunk 5]
```diff
@@ -161,21 +161,19 @@ public void registerSwagger(String microserviceId, List<String> schemaIds) {
...
+    temSwagger.getPaths().forEach((k, v) -> v.getOperations().forEach(method -> {
+      String processOptId = method.getOperationId();
+      processOptId = processOptId.substring(0, processOptId.indexOf("Using"));
+      if (methodFilter.contains(processOptId)) {
+        abandonedList.add(k);
+        return;
+      }
+      methodFilter.add(processOptId);
+      method.setOperationId(processOptId);
+    }));
```
Comment (GuoYL123) [inline spring-cloud-huawei-swagger/src/main/java/com/huaweicloud/swagger/ServiceCombSwaggerHandlerImpl.java]: the same as above

File: spring-cloud-huawei-swagger/src/main/java/com/huaweicloud/swagger/ServiceCombSwaggerHandlerImpl.java [hunk 6]
```diff
@@ -204,9 +202,7 @@ private void registerSwaggerSync(String microserviceId, List<String> schemaIds)
   }
 
   private void registerSwaggerAsync(String microserviceId, List<String> schemaIds) {
-    Executors.newSingleThreadExecutor().execute(() -> {
-      registerSwaggerSync(microserviceId, schemaIds);
-    });
+    Executors.newSingleThreadExecutor().execute(() -> registerSwaggerSync(microserviceId, schemaIds));
```
Comment (GuoYL123) [inline spring-cloud-huawei-swagger/src/main/java/com/huaweicloud/swagger/ServiceCombSwaggerHandlerImpl.java]: the same as above

File: spring-cloud-huawei-swagger/src/main/java/com/huaweicloud/swagger/ServiceCombSwaggerHandlerImpl.java (hunk 4 above)
Comment (AngLi2) [reply spring-cloud-huawei-swagger/src/main/java/com/huaweicloud/swagger/ServiceCombSwaggerHandlerImpl.java]: Idea suggests to delete { } for single statement in anonymous functions to simplify code block. The { } in lambda from lambda is not as meaningful as those in condition or loop statements. However, I think either is fine in this situation.

Comment (GuoYL123) [reply spring-cloud-huawei-swagger/src/main/java/com/huaweicloud/swagger/ServiceCombSwaggerHandlerImpl.java]: remove {} and don't put multiple loops on one line, seems easier to read

File: spring-cloud-huawei-swagger/src/main/java/com/huaweicloud/swagger/ServiceCombSwaggerHandlerImpl.java (hunk 5 above)
Comment (AngLi2) [reply spring-cloud-huawei-swagger/src/main/java/com/huaweicloud/swagger/ServiceCombSwaggerHandlerImpl.java]: done

File: spring-cloud-huawei-swagger/src/main/java/com/huaweicloud/swagger/ServiceCombSwaggerHandlerImpl.java (hunk 4 above)
Comment (AngLi2) [reply spring-cloud-huawei-swagger/src/main/java/com/huaweicloud/swagger/ServiceCombSwaggerHandlerImpl.java]: done

File: spring-cloud-huawei-swagger/src/main/java/com/huaweicloud/swagger/ServiceCombSwaggerHandlerImpl.java (hunk 6 above)
Comment (AngLi2) [reply spring-cloud-huawei-swagger/src/main/java/com/huaweicloud/swagger/ServiceCombSwaggerHandlerImpl.java]: done

**Pull request merged**